index.html

```diff
@@ -36,14 +36,14 @@ <h1 class="top-nav__left">Shia LaBeouf</h1>
         </div>
       </div>
     </nav>
-    <div class="section">
-      <section class="section__heading" id="welcome">
+    <div class="app__content">
+      <section class="area-of-control area-of-control--zapp" id="welcome">
```

Comment:
Looks like you figured out that `zapp` becomes `hero`. So now just work on looking at what would replace `area-of-control`